tracer/src/Datadog.Trace/IAST/Ranges.cs

```diff
@@ -6,6 +6,7 @@
 #nullable enable
 
 using System;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.OpenTelemetry;
```

Comment:
It seems unlikely this is the right namespace 🤔

Reply:
Yes, It is not actually used. I wonder how it got there...